test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": "00"
+    },
+    {
+      "dec": "1",
+      "hex": "01"
+    },
+    {
+      "dec": "-1",
+      "hex": "ff"
```

Comment:
@dcousens HEX is nothing special to `.toString()`, the base-36 or base-3 encoding should work the same way as the HEX does.

I guess the two's complement thing should happen in the code responsible for encoding this. There are various way to encode it, and this doesn't look like a reason to make HEX something special.